### PR TITLE
fix: grammar and spacing

### PR DESCRIPTION
**Pull request by crooruhe**
### Why:
Was reading documentation for oauth and ran across some confusing language
Fixes grammar and spacing

Closes:
N/A

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Fixes grammar/spacing in liquid template tag

### Check off the following:

- [ ] A subject matter expert (SME) has reviewed the technical accuracy of the content in this PR. In most cases, the author can be the SME. Open source contributions may require a SME review from GitHub staff.
- [x] The changes in this PR meet [the docs fundamentals that are required for all content](http://docs.github.com/en/contributing/writing-for-github-docs/about-githubs-documentation-fundamentals).
- [x] All CI checks are passing.

### CODE REVIEW

File: data/reusables/apps/deprecating_auth_with_query_parameters.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 > [!WARNING]
-> **{% data variables.release-phases.retired_caps %} Notice:** {% data variables.product.prodname_dotcom %} no longer to the API using query parameters. Authenticating to the API should be done with [HTTP basic authentication](/rest/overview/authenticating-to-the-rest-api#using-basic-authentication).For more information, including scheduled brownouts, see the [blog post](https://developer.github.com/changes/2020-02-10-deprecating-auth-through-query-param/).
```
Comment (jokerputihboy) [inline data/reusables/apps/deprecating_auth_with_query_parameters.md:2]: - [ ] - SURY BOYABOY

**Review by jokerputihboy — COMMENTED** [review]
JOKERBOY

**Comment by 6750903177** [comment]
> ### Why:
> Was reading documentation for oauth and ran across some confusing language Fixes grammar and spacing
> 
> Closes: N/A
> 
> ### What's being changed (if available, include any code snippets, screenshots, or gifs):
> Fixes grammar/spacing in liquid template tag
> 
> ### Check off the following:
> * [/#4310:6750903177:158.62.96.151]  A subject matter expert (SME°Aguinaldok4CLi/.) has reviewed the technical accuracy of the content in this PR. In most cases, the author can be the SME. Open source contributions may require a SME review from GitHub staff.
> * [ i@6750903177‽miui/`§]  The changes in this PR meet [the docs fundamentals that are required for all content](http://docs.github.com/en/contributing/writing-for-github-docs/about-githubs-documentation-fundamentals).
> * [6750903177//],  All CI checks are passing.

**Review by nguyenalex836 — APPROVED** [review]

**Comment by nguyenalex836** [comment]
@crooruhe Thanks so much for opening a PR with this fix! I'll update the branch and get this merged once tests are passing 🍏

**Pull request merged**